templates/inspec/tests/compute_instance_test.rb

```diff
@@ -0,0 +1,116 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'google_compute_instance'
+
+class InstanceTest < Instance
+  def initialize(data)
+    @fetched = data
+  end
+end
+
+instance_fixture = {"kind"=>"compute#instance",
```

Comment:
Idea: what if you take this JSON block and move it to a separate file? (Maybe even convert to YAML but I have less feelings on that)

It would clean up this test file a lot and make it easier to differentiate between fixtures + tests. Bonus - in the future, we could get to a world where we can more easily swap out the fixtures to ensure that they are up-to-date with whatever GCP is returning.